### DISCUSSION TITLE
6700 XT Support

### DISCUSSION THREAD

**Discussion by crusherv9** [jef/streetmerchant#2082] [Ideas and feature requests]
Hello,

would it be possible to get an update before RX 6700 XT release, so we can monitor it? Did anybody get the links or knows how you can configure this?

Best regards

**Comment by CamilloCorleone** [comment]
I was about to ask the same question, I'm following this request.

**Comment by damiankrzyz** [comment]
same for me please

**Comment by mshcb** [comment]
Hey,
it's quite not that hard. 

1. go to streetmerchant\src\store\model
2. open the store-file where you wan't to monitor it
3. change the url from any other graphics card and monitor it

**Comment by Galantha** [comment]
> Hello,
> 
> would it be possible to get an update before RX 6700 XT release, so we can monitor it? Did anybody get the links or knows how you can configure this?
> 
> Best regards

It is a bit of a pain, but you should do:
_file config.ts:_
add:  
`rx6700xt: envOrNumber(process.env.MAX_PRICE_SERIES_RX6700XT),`
to maxPrice

add:  
`'rx6700xt',`
to showOnlySeries

add: 
`rx6700xt: envOrArray(process.env.DISCORD_NOTIFY_GROUP_RX6800XT),`
to discord


_file store.ts_
add   
`| 'rx6700xt'`
to export type Series


_file amd.ts_
add:
```
    {
      brand: 'amd',
      cartUrl: 'https://www.amd.com/en/direct-buy/5496921400/us?add-to-cart=true',
      model: 'amd reference',
      series: 'rx6700xt',
      url: 'https://www.amd.com/en/direct-buy/5496921400/us',
    },
```


_file dotenv_
add:
```
MAX_PRICE_SERIES_RX6700XT=
DISCORD_NOTIFY_GROUP_RX6700XT=
```

**Reply by joeish** [reply]
> rx6700xt: envOrArray(process.env.DISCORD_NOTIFY_GROUP_RX6800XT)

Is this supposed to say rx6800xt?

**Reply by Galantha** [reply]
Looks like I made an error, it probably should say:
rx6700xt: envOrArray(process.env.DISCORD_NOTIFY_GROUP_RX6700XT)

however, I tested and used it with the above mentioned error, so I do not know if the corrected version works

**Comment by aradenac** [comment]
Hello, based on the Galantha's method, I initiated a [pull request ](https://github.com/jef/streetmerchant/pull/2290) to support the rx6700xt officially. You can see the changes in [the PR commit](https://github.com/jef/streetmerchant/pull/2290/commits/57d2350bf277b7b166ad853d8ae8b9fbcab3ef6f).

**Reply by jef** [reply]
Merged!
